### DISCUSSION TITLE
Transparent background

### DISCUSSION THREAD

**Discussion by cheetahbyte** [nvim-tree/nvim-tree.lua#1758] [Q&A]
Hello, I wanted to ask how I can make the background of the tree transparent. 
Can someone help me? :sweat_smile: 
![grafik](https://user-images.githubusercontent.com/40123243/202902241-27d4734a-e911-47ee-a593-aefa4ddd284c.png)

**Comment by alex-courtis** [comment]
We don't override the normal (background) of the tree window by default.

`NvimTreeNormal` and `NvimTreeNormalNC` defaut to `Normal` and `NormalNC`. What does `:so $VIMRUNTIME/syntax/hitest.vim` show for the `Normal`s?

How are you setting the background to be transparent?

Is `termguicolors` set?

**Reply by cheetahbyte** [reply]
This is what it shows me for `Normal`

![image](https://user-images.githubusercontent.com/40123243/203367951-9f49730e-4ecc-45c8-9686-33093246e2f0.png)

For `NvimTreeNormal`

![image](https://user-images.githubusercontent.com/40123243/203368208-32773518-0fd1-463c-ab77-a78ddc5ace16.png)

**Comment by fnacarellidev** [comment]
I'm having the same problem, when I open nvim-tree I lose the transparency background on the current file, I need to reopen it, I also wanted to make my nvim-tree transparent.
[Here's what's going on](https://imgur.com/a/hEdLKPt)

**Comment by alex-courtis** [comment]
> This is what it shows me for Normal

It looks like the link from NvimTreeNormal to Normal has been broken. It should be set correctly on a colorscheme event and to an already applied colorscheme.

Possible cause: something is (re)setting your highlight groups individually, outside of a colorscheme.

Instead of digging into the root cause you could work around it via `:hi link NvimTreeNormal Normal`. That should implicitly `NC` `SignColumn` etc.
